docs/migrating/migrating_from_ez_publish_platform.md

```diff
@@ -190,15 +190,24 @@ Steps here should only be done once you are ready to move away from legacy and L
 
 ###### 3.2.1 Migrate XmlText content to RichText
 
-**If** you are ready to migrate your eZ Publish XMLText content to the eZ Platform RichText format and start using pure eZ Platform setup, you can use a script to migrate content from the XmlText format to the new RichText format. Execute the following from &lt;new-ez-root&gt;:
+You should test the XmlText to RichText conversion before you apply it to a production database. RichText has a stricter validation compared to XmlText and you may have to fix some of your XmlText before you are able to convert it to RichText.
+Run the conversion script on a copy of your production database as the script is rather resource-intensive.
 
-`php app/console ezxmltext:convert-to-richtext -v`
+`php -d memory_limit=1512M bin/console ezxmltext:convert-to-richtext --dry-run --export-dir=ezxmltext-export --export-dir-filter=notice,warning,error --concurrency 4 -v`
 
-The command example suggests using the verbose flag. This is optional, but recommended as we are actively gathering feedback on how it works with older eZ Publish content, and on how we can improve it both [via issues in the repository](https://github.com/ezsystems/ezplatform-xmltext-fieldtype), and on Slack.
+- `-d memory_limit=1500M` specifies that each conversion process gets 1500MB of memory. This should be more than sufficient for most databases. If you have small `ezxmltext` documents, you may decrease the limit. If you have huge `ezxmltext` documents, you may need to increase it.
```

Comment:
command uses 1512, not 1500, but if the intention is to use 1.5GB it should be 1536 :)

Either way, besides making it consistent, please add link to http://php.net/manual/en/ini.core.php#ini.memory-limit  on for instance "1500MB of memory" for further reading in cases reader is new to this

Reply:
Changed to 1536 everywhere and add link to memory_limit documentation